intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -1516,6 +1561,33 @@ def convert_date_utc(value: str) -> Optional[str]:
     },
 }
 
+accessible_rdp_eudp = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port'),
+    ],
+    'optional_fields': [
+        ('source.reverse_dns', 'hostname'),
+        ('classification.identifier', 'tag'),  
```

Comment:
that gets overwritten by the mapping in the constant fields below

Reply:
will rewrite that part of the code